### DISCUSSION TITLE
Higher land/water/build limits

### DISCUSSION THREAD

**Discussion by MaxMallon** [OpenRCT2/OpenRCT2#16049] [Feature requests]
During Marcel's interview stream last Thursday, I gave this suggestion/question. 
I'd quite like higher build limits. 
You said it'd be easy to implement higher land and water limits within the overal maximum height limit. I think that'de be great, cause it allows more dramatic landscapes, especially now that the maps themselves can be bigger too. 
I often have a tall map with mountains terrain, and I think "Oh, let's add a little pond here", and the game's like NOPE, and I'm like "why not?! D:"

If you'd also increase the height field itself, to allow taller build height for everything, like coasters and paths, that'd be absolutely amazing. I run into that limit all the time, with things like drop towers, and tall coasters on top of mountains. A 300 meter vertical drop coaster would be just 11/10

**Comment by J-PIE-314** [comment]
The land and water height limits have been raised. (https://github.com/OpenRCT2/OpenRCT2/pull/19988) Support limits have also been adjusted accordingly.
